pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -322,4 +322,5 @@ message IndexBackfillMergerSpec {
   repeated int32 span_idx = 5;
 
   optional int64 chunk_size = 6 [(gogoproto.nullable) = false];
+  optional int64 chunk_bytes = 7 [(gogoproto.nullable) = false];
```

Comment:
in retrospect I don't think it's a great idea to put either of these in the spec. Instead, can we just have each of the processors read from the setting per batch? That way you could change it at runtime without a pause-resume cycle. Just a suggestion.

Reply:
Done, implemented reading the setting at beginning of each batch.